cartpole/cartpole.jl

```diff
@@ -0,0 +1,92 @@
+using Reinforce:CartPole, actions, reset!, Episode, finished
+import Reinforce.action
+using Flux, StatsBase, Plots
+
+gr()
+
+#Define custom policy for choosing action
+type CartPolePolicy <: Reinforce.AbstractPolicy end
```

Comment:
`mutable struct` ?

Reply:
For the CartPolePolicy? 
Also, would like to know the advantage that would offer. AFAIK (and I may be wrong, as I am new to Julia), `mutable struct` for that would not have any fields.

Reply:
```julia
mutable struct CartPolePolicy <: Reinforce.AbstractPolicy end
```

Keyword `type` is deprecated syntax since Julia 0.6.

Reply:
Oh, I see. Thanks for that. I'll change it.